tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -192,3 +192,10 @@ def log_autolog_called(self, integration, call_args, call_kwargs):
     call = universal_autolog_event_logging_calls[0]
     assert call.integration == "mlflow"
     assert {"disable": True, "exclusive": True}.items() <= call.call_kwargs.items()
+
+
+def test_autolog_success_message_suppressed_when_disabled():
+    with mock.patch("mlflow.tracking.fluent._logger.info") as autolog_logger_mock:
```

Comment:
Can we extend this test case to test both the enable & disable path? I'd like to make sure that we're mocking the correct logger here (otherwise, this test case will pass vacuously).

Reply:
Yup! I did manually verify that the test fails without the change. But good to make it explicit

Reply:
Looks gr9!